demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -351,15 +351,15 @@ def is_changed_subtype(self):
     def is_valid_beta(self):
         # type: () -> bool
         """Validate that that beta integration has correct beta attributes"""
-
+        valid_status = True
         if not all([self._is_display_contains_beta(), self._has_beta_param()]):
             self.is_valid = False
-            return False
-        if self.old_file:
+            valid_status = False
+        if not self.old_file:
             if not all([self._id_has_no_beta_substring(), self._name_has_no_beta_substring()]):
                 self.is_valid = False
-                return False
-        return True
+                valid_status = False
+        return valid_status
```

Comment:
I think that after adding the `not self.old_file` this should work, so why do we need the `valid_status` argument?

Reply:
I want to return true or false only at the end of the function so that it passes all the tests and does not return in the first failure

Reply:
Mmmmm... why? if the first check fails the status will be `False` no matter what will happen in the second check. Is there something else happening in the second check that I am missing?

Reply:
Yes, inside the functions it puts the error into the handle_error and it prints the error, and if it finishes in the first function then only the error of the first will return

Reply:
Cool I see